WooCommerce/Classes/Notifications/LocalNotification.swift

```diff
@@ -130,51 +101,10 @@ extension LocalNotification {
 
 extension LocalNotification {
     enum Localization {
-        enum StoreCreationComplete {
-            static let title = NSLocalizedString(
-                "🎉 Your store is ready!",
-                comment: "Title of the local notification about a newly created store"
-            )
-            static let body = NSLocalizedString(
-                "Hi %1$@, Welcome to your 14-day free trial of Woo Express – " +
-                "everything you need to start and grow a successful online business, " +
-                "all in one place. Ready to explore?",
-                comment: "Message on the local notification about a newly created store." +
-                "The placeholder is the name of the user."
-            )
-        }
-
-        enum SixHoursAfterFreeTrialSubscribed {
-            static let title = NSLocalizedString(
-                "🌟 Keep your business going!",
-                comment: "Title of the local notification to remind the user to purchase a plan."
-            )
-            static let body = NSLocalizedString(
-                "Discover advanced features and personalized recommendations for your store! Tap to pick a plan that suits you best.",
-                comment: "Message on the local notification to remind the user to purchase a plan."
-            )
-        }
-
-        enum FreeTrialSurvey24hAfterFreeTrialSubscribed {
-            static let title = NSLocalizedString(
-                "💡Help Us Understand Your Subscription Decision",
-                comment: "Title of the local notification to ask for Free trial survey."
-            )
-            static let body = NSLocalizedString(
-                "We’re interested in your decision-making journey. Could you please tell us about your current status?",
-                comment: "Message on the local notification to ask for Free trial survey."
-            )
-        }
-
-        enum ThreeDaysAfterStillExploring {
-            static let title = NSLocalizedString(
-                "🧭 Still Exploring WooCommerce?",
-                comment: "Title of the local notification to remind after three days."
-            )
-            static let body = NSLocalizedString(
-                "No rush, take your time! If you have any questions or need assistance, we're always here to help. Happy exploring!",
-                comment: "Message on the local notification to remind after three days."
-            )
-        }
+        static let unknown = NSLocalizedString(
+            "localNotification.Unknown",
+            value: "Unknown",
+            comment: "A placeholder value for local notification"
+        )
```

Comment:
We should not need to add this for translation, simply setting the title and body to an empty string should be sufficient.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/12443/commits/ca91d2429c52d062973c89afb2944f84ef95788f